corehq/apps/users/urls.py

```diff
@@ -67,7 +67,10 @@
         'user_upload_job_poll', name='user_upload_job_poll'),
     url(r'^commcare/download/$', 'download_commcare_users', name='download_commcare_users'),
     url(r'^commcare/set_group/$', 'set_commcare_user_group', name='set_commcare_user_group'),
-    url(r'^commcare/add_commcare_account/$', CreateCommCareUserView.as_view(), name=CreateCommCareUserView.urlname),
+    url(r'^commcare/add_commcare_account/$', CreateCommCareUserView.as_view(),
+        name=CreateCommCareUserView.urlname),
+    url(r'^commcare/add_commcare_account/(?P<location_id>[\w-]+)/$', CreateCommCareUserView.as_view(),
+        name=CreateCommCareUserView.urlname),
```

Comment:
small point, but I think I would prefer if you used a query param for this, since that would be a more reasonable way to specify defaults for a bunch of different parameters down the road (as opposed to having to make a new url mapping for each one)